mlir/test/mlir-miopen-driver/e2e/conv2d_host_validation_f16.mlir

```diff
@@ -115,6 +115,12 @@
 // CHECK_BWD_DATA1: Unranked Memref base@ = 0x{{.*}} rank = 1 offset = 0 sizes = [1] strides = [1] data =
 // CHECK_BWD_DATA1: [1]
 
+
+// COM: bwd_data
```

Comment:
@ltqin the case above this one has dilation and stride both equal to 1, can its FIXME be changed to RUN?

Reply:
done